master/cmd/determined-master/init.go

```diff
@@ -129,4 +130,9 @@ func registerConfig() {
 		defaults.Telemetry.SegmentMasterKey, "the Segment write key for the master")
 	registerString(flags, name("telemetry", "segment-webui-key"),
 		defaults.Telemetry.SegmentWebUIKey, "the Segment write key for the WebUI")
+
+	registerString(flags, name("checkpoint-storage", "type"),
```

Comment:
this should work fine afaik, but, you know, test it, lol. i'm not 100% sure how this `registerString` will work with our union types.

Reply:
I've had to register this type (and not just host_path) because the union type validator would throw up on host_path without type=shared_fs.